src/identity/contactMapping.ts

```diff
@@ -207,6 +211,34 @@ export function* fetchAddressesAndValidateSaga({
   }
 }
 
+export function* fetchAddressVerificationSaga({ address }: FetchAddressVerificationAction) {
+  try {
+    const verifiedAddresses = yield* select(verifiedAddressesSelector)
+    if (verifiedAddresses.includes(address)) {
+      ValoraAnalytics.track(IdentityEvents.address_lookup_skip)
+    } else {
+      ValoraAnalytics.track(IdentityEvents.address_lookup_start)
+      const addressVerified = yield* call(fetchAddressVerification, address)
+      if (addressVerified) {
+        yield* put(updateVerifiedAddresses([...verifiedAddresses, address]))
+      }
+      ValoraAnalytics.track(IdentityEvents.address_lookup_complete)
+    }
+  } catch (err) {
+    const error = ensureError(err)
+    Logger.debug(
```

Comment:
should this be a higher log level? or could it be noisy w/ rate limiting stuff?

Reply:
addresses in other comment